orttraining/orttraining/test/gradient/gradient_ops_test.cc

```diff
@@ -546,6 +546,74 @@ TEST(GradientCheckerTest, ReduceSumGrad) {
   }
 }
 
+TEST(GradientCheckerTest, ReduceLogSumExpGrad) {
```

Comment:
Follow up: I think we can refactor the test code and make a shared function for ReduceSum, ReduceMean and ReduceLogSumExp

Reply:
refactored the code.